frontend/src/components/camp/CollaboratorListItem.vue

```diff
@@ -18,49 +18,92 @@
         color="normal"
         icon="mdi-refresh"
         :animate="resendingEmail"
+        :disabled="disabled"
         @click="resendInvitation">
         {{ $tc('components.camp.collaboratorListItem.resendEmail') }}
       </icon-button>
     </v-list-item-action>
     <v-list-item-action class="ml-4">
-      <api-select
-        :value="collaborator.role"
-        :uri="collaborator._meta.self"
-        fieldname="role"
-        :items="[
-          { key: 'member', translation: $tc('entity.camp.collaborators.member') },
-          { key: 'manager', translation: $tc('entity.camp.collaborators.manager') },
-        ]"
-        item-value="key"
-        item-text="translation"
-        :my="0"
-        dense
-        vee-rules="required" />
+      <v-tooltip :disabled="disabled || !isLastManager" top>
+        <template #activator="{ on, attrs }">
+          <div
+            v-bind="attrs"
+            v-on="on">
```

Comment:
Die direkt aufs `api-select` bzw. den button-delete zu machen funktioniert nicht?

Reply:
Leider nein, weil die disablen sind in dem fall, in dem das popup angezeigt wird.